src/main/java/org/reservationapplication/CoworkingSpace.java

```diff
@@ -0,0 +1,54 @@
+package org.reservationapplication;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CoworkingSpace {
+    private static long nextId = 0L;
+    private final long coworkingSpaceID;
+    private CoworkingSpaceType type;
+    private double price;
+    private AvailabilityStatus availabilityStatus;
+
+    public CoworkingSpace(CoworkingSpaceService coworkingSpaceService) {
+        this.coworkingSpaceID = nextId++;
```

Comment:
no, coworking space does't have to know about own service. Even repository here will be tough.